modules/3d-tiles/src/lib/utils/s2/s2-geometry.ts

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
Is this copied from deck.gl or is it an independent effort? Seeing this in multiple places, starting to feel like it is time to make an `@math.gl/s2` module. @Pessimistress what do you think about that?